tests/test_file_handler.py

```
@@ -45,13 +43,27 @@ def send(h: FemtoFileHandler, i: int) -> None:
         t.start()
     for t in threads:
         t.join()
-    del handler
-    gc.collect()
+    handler.close()
     data = path.read_text()
     for i in range(10):
         assert f"core [INFO] msg{i}" in data
 
 
+def test_file_handler_flush(tmp_path: Path) -> None:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)